apps/tailwind-components/app/components/table/TableEMX2.vue

```diff
@@ -277,8 +277,8 @@ const rows = computed(() => {
   return data.value.tableData.rows;
 });
 
-const showDraftColumn = computed(() =>
-  rows.value.some((row) => row?.mg_draft === true)
+const showDraftColumn = computed(
+  () => rows.value?.some((row) => row?.mg_draft === true) || true
```

Comment:
?

Reply:
ik heb alleen de ? toegevoegd op rows.value? omdat als bepaalde schemas niet bestonden de demo pages niet wilden laden

Reply:
The expression always evaluates to true.

Reply:
ah, you pushed a better solution I noticed, thx